### ISSUE TITLE
Wyze plug

### ISSUE THREAD

**Issue by Coltuxumab** [kevinvincent/ha-wyzesense#187]
Tried searching but couldn’t find this request.

I know the Wyze plug isn’t technically part of Wyze sense, but would it be possible to add support?

**Comment by jamescadd** [comment]
I would doubt that, it's wifi and wouldn't use that hub. I looked at some briefly and the connections to flash it internally are a pain to solder, would recommend you look at Guys convert over the air (personally I prefer ESPHome, but the same tool can flash it as well).

**Comment by Coltuxumab** [comment]
Ah now I see, you’re right. Damn, I thought this would work as well as the sensors do. Makes sense, thanks for the quick reply.

**Comment by photinus** [comment]
Check out this HA component: https://github.com/JoshuaMulliken/ha-wyzeapi

**Comment by Coltuxumab** [comment]
> Check out this HA component: https://github.com/JoshuaMulliken/ha-wyzeapi
> […](#)

Yeah I did see that, but it just uses your wyze login info to connect. My goal was to avoid the cloud like the other sensors. I can see now that the plug does not work the same way as the sensors (returned it).

**Comment by VajraCool** [comment]
Is there anyway can this make local ? anyone have links for Flashing? I hope it is easy with OTA
